### PR TITLE
Guys checkout my new shubhart

### PR DESCRIPTION
**Pull request by shubhamuni**
hello @aneagoie  checkout my new css animation

### CODE REVIEW

**Comment by shubhamuni** [comment]
please add

File: Art/shubhart/style.css
Comment (LaurelineP) [inline Art/shubhart/style.css:1]: ## Commented code
Please remove the commented codes

File: include.js
Comment (LaurelineP) [inline include.js:1]: ## Incorrect folder name
The folder name should be named after you github username - please adjust :
- the folder name
- your include.js contribution to adjust the paths

**Review by LaurelineP — CHANGES_REQUESTED** [review]
Aloha 🙌
Thanks for your contribution and time !
Before we could merge your contribution, the following feedbacks need to be addressed first.
Please address the given feedbacks

> [!NOTE]
Checked values - are the one that are correctly done
> - Please address all the unchecked elements
> - If you need guidance , please refer to the

> [!TIP]
> If the name of your folder needs to be changed ensure to adjust the `include.js`

## Feedbacks - Missing CSS files in your contributions
## 🔹 Reference Acceptance criteria for a contribution
Correctly:
  - [ ] Folder name: named after the github user name in `./Art`  
     - [x] contains a `HTML` file: the contribution must have an HTML file
         - [x] has a <link> tag to the CSS file of the project
         - [x] contains the `HTML` content for the animation
         - [x] does not contain a SVG replacing the whole HTML CSS animation requirements
     - [ ] contains a `CSS` file: the contribution must have a CSS file 
     - [x] does not contain a `JS` file or JS logic : No JS logic should be part of your submission
     - [x] the contribution includes the animated file (gif) or picture ( jpeg, jpg, png )
     - [x] the pictural representation must not include the browser frame, just the animation
     - [x] the contribution includes the correct contribution's details in `include.js`
        - [ ]  the relative path should be correct : referring to the correct named folder
        - [x] `author`: the value should be the github name or github
        - [x] `artName`: the value should be the animation title
        - [ ] `pageLink`: the value should  be the relative path to your project's `HTML` file
        - [ ] `imageLink`: should refer to the project relative path's image 
        - [x] `githubLink`: the value should be your github profile link
        - [x] does not have chevrons
        - [x] does not have comments for each fields

**Comment by shubhamuni** [comment]
Please check and confirm

**Pull request closed without merging**